third_party/eigen3/BUILD

```diff
@@ -28,6 +30,7 @@ cc_library(
         "unsupported/Eigen/CXX11/Tensor",
         "unsupported/Eigen/CXX11/FixedPoint",
     ],
+    includes = if_mkl(["./mkl_include"]),
     visibility = ["//visibility:public"],
 # INTEL_MKL start
 		includes = if_mkl(["./mkl_include"]),
```

Comment:
the includes= is repeated again here.  I had removed the INTEL_MKL_start/end comments.